requirements.md

```diff
@@ -91,6 +114,8 @@ authorization it can be validated against relying-party stored instrument displa
 
 * For privacy, the protocol must enable the user to require that two payment instruments NOT be associated with the same authenticator.
 
+Q: Please clarify.
```

Comment:
I am removing this one for now. And I closed issue 43 which I think was poorly expressed in any case.

Here's the goal I have in mind:

* If a person has two instruments and has done what's necessary so that merchants paid with those instruments do not have enough information to correlate them to the user, then the web protocol itself should not provide information to enable correlation.